sdk/javascript/generator/PodTypeFormatter.py

```diff
@@ -43,6 +43,14 @@ def get_deserialize_descriptor(self):
 		body += f'return new {self.typename}({self.printer.load()});'
 		return MethodDescriptor(body=body)
 
+	def get_deserialize_aligned_descriptor(self):
+		if self._is_array:
+			return None
```

Comment:
a) this only means that deserializeAligned won't be generated for types like Address, Hash256 etc... mostly cause it doesn't make much sense and it would have to be exactly same as deserialize

what deserialize() for ByteArray does is:
 * it creates "shallow" uint8array around (offset, buffer, size)
 * it then passes that shallow reference to ctor, which then copies it 
 
technically we could _try_ to optimize that, although I don't think it's _worth_ it (moreover, I assume there's some sensible copying happening under the hood)